code/sorting/src/sleep_sort/sleep_sort.c

```diff
@@ -9,19 +9,20 @@
 #include <windows.h>
 void routine(void *a)
```

Comment:
Return types go on their own line (style).

Reply:
didn't understand what you meant. Can you give an example?

Reply:
@arnavb did you mean add function prototypes?

Reply:
@yedhink Our [C style guide](https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c) mandates that return types go on their own line:

```C
void
function() {
<TAB>...code...
}
```
Make sure to use tabs, not spaces as well.

Reply:
@arnavb i thought we were adopting the the linux kernel c style guide as discussed in #3947. I have followed the kernel c style guide. Wont we have to again reformat the code when adopting the new style if we follow the c style guide that you mentioned?

Reply:
also i have been developing scripts to autoformat to the linux kernel style guide with the exception of using 4 spaces as mentioned in #3947. So currently changing the code back to the c style you mentioned would be a mess. 

Is it absolutely mandatory to change?

Reply:
@yedhink Until the autoformatting script gets merged, please follow our current style guide, because there is no 100% guarantee it will be implemented yet either.

Reply:
@arnavb have made the necessary changes. see if its okay.